wallet/src/main/scala/org/bitcoins/wallet/config/WalletAppConfig.scala

```diff
@@ -73,15 +73,18 @@ case class WalletAppConfig(
   }
 
   /** The path to our encrypted mnemonic seed */
-  private[wallet] def seedPath: Path = {
-    datadir.resolve(WalletAppConfig.ENCRYPTED_SEED_FILE_NAME)
+  private[bitcoins] def seedPath: Path = {
```

Comment:
I assume this is done so both the `wallet` and `keymanager` projects can access this, its a shame that you can't list packages